ci_framework/roles/hive/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Fails if the oc client is lesser than 4
+- name: Ensuring the supported version of oc is present
+  environment:
+    PATH: "{{ cifmw_path }}"
+    ansible.builtin.shell:
+      cmd: oc version -o json | jq -r .releaseClientVersion | cut -d '.' -f 1
+    register: oc_version_result
+    changed_when: false
+    failed_when: ( oc_version_result.stderr | length > 0 ) or
+                 ( oc_version_result.stdout | int ) < 4
+
```

Comment:
IMO, we could ensure the artifacts/output directory exists
```
- name: Ensure hive output directory exists
  ansible.builtin.file:
    path: "{{ cifmw_hive_artifacts_dir }}"
    state: directory
```

We would need to set in `defaults/main.yml` the value for `cifmw_hive_artifacts_dir`